galleries/examples/images_contours_and_fields/specgram_demo.py

```diff
@@ -1,9 +1,9 @@
 """
-================
-Spectrogram Demo
-================
+===========
+Spectrogram
+===========
 
-Demo of a spectrogram plot (`~.axes.Axes.specgram`).
+Plotting a spectrogram using `~.Axes.specgram` in Matplotlib.
```

Comment:
Yes, I know how to do that. However, some of them used it originally and I guess it will give (slightly) better search engine hits? (And a bit more clarity when meta sites copy the examples etc.)